recipes/rmlui/3.3/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+
+if(NOT BUILD_SHARED_LIBS)
+    target_compile_definitions(example PRIVATE RMLUI_STATIC_LIB)
+endif()
```

Comment:
This needs to be set in `package_info()` like 

https://github.com/conan-io/conan-center-index/blob/689e2663fe41689259f61a5dbd3fe245e24c03b9/recipes/nng/all/conanfile.py#L86